### PR TITLE
EPMRPP-90207 || Sidebar should be expanded on hovering and collapsed …

### PR DESCRIPTION
**Pull request by BlazarQSO**
…on unhovering

## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/layouts/common/appSidebar/appSidebar.jsx [hunk 1]
```diff
@@ -55,13 +55,14 @@ export const AppSidebar = ({
...
 
-  const createFooterBlock = (openNavbar) => (
+  const createFooterBlock = (openNavbar, onImmediatelyOpenSidebar) => (
     <>
       <div className={cx('policy-block')} />
       <UserAvatar
         onClick={() => {
           openNavbar();
           setIsOpenAvatarPopover(true);
+          onImmediatelyOpenSidebar();
```
Comment (Vadim73i) [inline app/src/layouts/common/appSidebar/appSidebar.jsx:65]: <Optional> May be better move it to openNavbar and call inside?
openNavbar(onImmediatelyOpenSidebar)

Comment (BlazarQSO) [reply app/src/layouts/common/appSidebar/appSidebar.jsx:65]: These are different functions responsible for different events

Comment (BlazarQSO) [reply app/src/layouts/common/appSidebar/appSidebar.jsx]: This behavior was removed

File: app/src/componentLibrary/sidebar/navbar/navbar.scss [hunk 2]
```diff
@@ -35,6 +36,10 @@
   transform: translateX(0);
 }
 
+.quickly {
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/componentLibrary/sidebar/navbar/navbar.scss]: Done.

**Review by AmsterGet — APPROVED** [review]

**Review by Vadim73i — APPROVED** [review]

**Pull request merged**